From 7ca47b7c46f3e8ba2be9c33c31e7f731338bf236 Mon Sep 17 00:00:00 2001
From: Mandy Lavi <mandy.lavi@freescale.com>
Date: Tue, 9 Jul 2013 14:36:48 +0300
Subject: [PATCH 324/987] fmd: COMPAT array size increased in order to be able
 to map more ccnode ids

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Signed-off-by: Mandy Lavi <mandy.lavi@freescale.com>

ENGR00270413

Change-Id: I9ca855698d1eaa93b7cb131d811f4c9dd6c72088
Reviewed-on: http://git.am.freescale.net:8181/3297
Reviewed-by: Bucur Madalin-Cristian-B32716 <madalin.bucur@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 .../net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h
index 288777a..f8655db 100644
--- a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h
+++ b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h
@@ -52,7 +52,7 @@
 /* mapping kernel pointers w/ UserSpace id's { */
 /* Because compat_ptr(ptr_to_compat(X)) != X, this way we cannot exchange pointers
    back and forth (US - KS). compat_ptr is a cast and pointers are broken. */
-#define COMPAT_PTR2ID_ARRAY_MAX (256+1) /* first location is not used */
+#define COMPAT_PTR2ID_ARRAY_MAX (512+1) /* first location is not used */
 #define COMPAT_PTR2ID_WATERMARK 0xface0000
 #define COMPAT_PTR2ID_WM_MASK   0xffff0000
 
-- 
1.9.1

